### ISSUE TITLE
ERROR deprecated: .Site.GoogleAnalytics was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.133.0. Use .Site.Config.Services.GoogleAnalytics.ID instead.

### ISSUE THREAD

**Issue by ShaharNaveh** [imfing/hextra#433]
Since upgrading to Hugo 0.132.0, I get the following error:

```
ERROR deprecated: .Site.GoogleAnalytics was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.133.0. Use .Site.Config.Services.GoogleAnalytics.ID instead.
```

That's because I treat warnings as errors in my build process.

---

I've found the following places that tries to access `.Site.GoogleAnalytics`:

- https://github.com/imfing/hextra/blob/e83c11f31a00c1088cd49e8eaea10b60384921c4/layouts/partials/google-analytics.html#L4
- https://github.com/imfing/hextra/blob/e83c11f31a00c1088cd49e8eaea10b60384921c4/layouts/partials/head.html#L34

The fix should be fairly simple; just removing the places where `.Site.GoogleAnalytics` is accessed.

I'll be more than glad to open a PR if that would get accepted, lmk what you think.

---

Thanks for the awesome theme!

**Comment by pcelka** [comment]
Same Error, your fix helped me !

**Comment by eskopp** [comment]
The issue is resolved with pull request #435.
